code/lib/manager-api/src/modules/stories.ts

```diff
@@ -102,6 +114,7 @@ export interface SubAPI {
   updateStory: (storyId: StoryId, update: StoryUpdate, ref?: API_ComposedRef) => Promise<void>;
   updateDocs: (storyId: StoryId, update: DocsUpdate, ref?: API_ComposedRef) => Promise<void>;
   setPreviewInitialized: (ref?: ComposedRef) => Promise<void>;
+  experimental_updateStatus: (id: string, update: StatusUpdate) => Promise<void>;
```

Comment:
Is the `id` here the addon id? that's not obvious. I'd suggest changing the var name and considering giving it a special type.

Can we document the feature via a JSdoc comment too? I think we should get into the practice of doing that.